source/nodejs/adaptivecards-templating/src/template-engine.ts

```diff
@@ -215,10 +215,10 @@ export class Template {
      */
     public static parseInterpolatedString(interpolatedString: string): AEL.Expression | string {
         let lookup: AEL.EvaluatorLookup = (type: string) => {
-            let standardFunction = AEL.ExpressionFunctions.standardFunctions.get(type);
+            let matchedFunction = AEL.Expression.functions.get(type);
```

Comment:
Just to be sure as `Expressions.functions` didn't yet exists (as far as I remember) when I coded this originally - the `Expressions.functions` collection combines standard functions and custom ones, correct?

Reply:
Hey,

Thanks for the review, yep that's correct. It references a [FunctionTable instance](https://github.com/microsoft/botbuilder-js/blob/main/libraries/adaptive-expressions/src/functionTable.ts) which combines them both.